payment/views.py

```diff
@@ -26,3 +26,8 @@ def payment_list(request):
     }
 
     return render(request, 'payment/payment_list.html', context)
+
+def payment_delete(request, payment_id):
+    payment = Payment.objects.get(id=payment_id)
```

Comment:
To avoid possible errors of trying to access a payment that does not exist you can use the function _get_object_or_404_, like **_payment = get_object_or_404(Payment, id=payment_id)_**
To use this function it is necessary to import it as **_from django.shortcuts import get_object_or_404_**